scripts/update-nordix-repos-master.sh

```diff
@@ -115,4 +117,20 @@ done
 #  popd
 #  echo -e "\n"
 
-popd
+CAPM3_RELEASE_BRANCHES="release-0.3 release-0.4 release-0.5"
+i=0
+for branch in ${CAPM3_RELEASE_BRANCHES}
+do
+  echo "Updating ${branch} branch in ${LOCAL_CAPM3_REPO}"
+  pushd "${LOCAL_CAPM3_REPO}" || exit
+  git checkout "origin/${branch}"
+  git fetch "origin" "${branch}"
+  git rebase "origin" "${branch}"
+  git remote add "nordixrepo${i}" "${NORDIX_CAPM3_REPO}"
```

Comment:
could have been also```suggestion
  git remote add "nordixrepo${branch}" "${NORDIX_CAPM3_REPO}"
```
, but I'm fine with the current approach too

Reply:
I just wanted to avoid using special chars in the remote name e.g. " - " and  ".".

Reply:
But now I have tested it can handle special chars so I have changed it according to your suggestion.

Reply:
thanks